sos/collector/clusters/ocp.py

```diff
@@ -142,12 +142,30 @@ def setup(self):
             self.fmt_oc_cmd("new-project %s" % self.project)
         )
         if ret['status'] == 0:
+            self._label_sos_project()
             return True
 
         self.log_debug("Failed to create project: %s" % ret['output'])
         raise Exception("Failed to create temporary project for collection. "
                         "\nAborting...")
 
+    def _label_sos_project(self):
+        """Add pertinent labels to the temporary project we've created so that
+        our privileged containers can properly run.
+        """
+        labels = [
+            "security.openshift.io/scc.podSecurityLabelSync=false",
+            "pod-security.kubernetes.io/enforce=privileged"
+        ]
+        for label in labels:
+            ret = self.exec_primary_cmd(
+                self.fmt_oc_cmd(f"label namespace {self.project} {label}")
```

Comment:
this will complain `Error applying namespace labels: error: 'pod-security.kubernetes.io/enforce' already has a value (restricted), and --overwrite is false`
to fix you can add this flag
`self.fmt_oc_cmd(f"label namespace {self.project} {label} --overwrite")`

Reply:
With that change works well (but I guess if you didn't catch that error, the version of OCP is not new enough)
tested for 4.12 nightly